rotkehlchen/typing.py

```diff
@@ -114,8 +114,21 @@ def serialize_for_db(self) -> Tuple[str, str]:
 T_BTCAddress = str
 BTCAddress = NewType('BTCAddress', T_BTCAddress)
 
-BlockchainAddress = Union[EthAddress, BTCAddress, ChecksumEthAddress, str]
-ListOfBlockchainAddresses = Union[List[BTCAddress], List[ChecksumEthAddress]]
+KusamaAddress = NewType('KusamaAddress', str)
```

Comment:
I'd like to have them in the substrate `typing.py`, but it ends up triggering an import problem. It is not a priority now and may be I'm not right about having them there!

Reply:
It should probably be there. The import problem is I guess some kind of cyclic import? You end up importing this typing in kusama typing through some other import. These are stuff you have to learn to fix ;)